demisto_sdk/commands/test_content/test_modeling_rule/test_modeling_rule.py

```diff
@@ -248,8 +296,7 @@ def validate_expected_values(
         )
         logger.debug(query)
         try:
-            execution_id = xsiam_client.start_xql_query(query)
-            results = xsiam_client.get_xql_query_result(execution_id)
+            results = retrying_caller(xsiam_execute_query, xsiam_client, query)
```

Comment:
it will work, but that's not the expected behaviour i'm looking for.
i want the entire query transaction to be confined by the retry, and not the "single" request.